doc/lua_api.md

```diff
@@ -7118,6 +7118,9 @@ Defaults for the `on_place` and `on_drop` item definition functions
       the item to the picker's `"main"` inventory list.
     * Parameters are the same as in `on_pickup`.
     * Returns the leftover itemstack.
+* `core.item_secondary_use(itemstack, user)`
+    * Global secondary use callback. Does nothing.
+    * **Note**: is not called when wielded item overrides `on_secondary_use`
```

Comment:
Missing return value documentation.

Reply:
I would just refer to `on_secondary_use`

Reply:
This function might (in theory) have multiple return values, whereas the callback only uses/needs the first one. Other return values could - for example - give modders additional hints or context.
This function is trviial, and yet, I don't think that referring to the callback alone is clear enough.

Reply:
But a no-op function cannot possibly have any additional useful behaviors or parameters...